environment-dev.yml

```diff
@@ -5,10 +5,10 @@ channels:
   - nvidia
   - conda-forge
 dependencies:
-  - numpy>=1.20
+  - numpy>=1.25,<2.0
```

Comment:
Hi @ericspod, is there a specific reason to use numpy>=1.25? The base image for MONAI 1.4 will be PyTorch 24.08, which will continue to use numpy == 1.24.4.

Reply:
There wasn't except to use a more recent version than 1.20, so it can be set to 1.24.